### ISSUE TITLE
Devtools preview

### ISSUE THREAD

**Issue by iBobik** [vue-email/vue-email#190]
Just installed new version and I miss dev server preview in Nuxt Devtools.

Will it be included or there is some new undocumented preview solution?

**Comment by Flowko** [comment]
I'm planning to build something like react email preview, but i might also do a nuxt devtools preview that has extra features, im currently busy with work, so it'll take some time.

**Comment by MickL** [comment]
A possibility to preview would be amazing! :)

Currently I render the mail and display it in an iframe

**Comment by BayBreezy** [comment]
Any plans to add this again? 
Can you provide a guide on how to add it locally please?

**Comment by Mokkapps** [comment]
For Nuxt users: I developed a new Nuxt module that provides a DevTools preview functionality: https://nuxtemail.com

**Comment by MickL** [comment]
@Mokkapps Is this another email library? Why didnt you contribute to vue-email instead?

**Comment by BayBreezy** [comment]
Thank you @Mokkapps 

@MickL there has been no update or sign of this feature being implemented. The last commit to main was last year November.
Sometimes we have to move on and create our own versions of packages like these. 
In another issue, help was offered to have this feature implemented and the author did not respond 🤷🏽‍♂️

**Comment by MickL** [comment]
@Mokkapps Did you make a fork or is this a complete new project?

If vue-email is no longer maintained I would have suggested to make a fork. If nuxt-email-renderer is a complete new project I think it is not a good idea to reinvent the wheel, have two separate projects that to the same thing and instead focus making one great open source solution. E.g. I dont see nuxt-email-renderer having a Tailwind integration that vue-email already had. Also how long is nuxt-email-renderer maintained? Maybe in a year it is not maintained anymore too and then?

You project seems to be targeting Nuxt only. I would say that it makes more sense to have a Vue-first library (like vue-email) that also has a Nuxt module instead of making it only for Nuxt. For example, Nuxt UI also made it a Vue library with v3.

**Comment by Mokkapps** [comment]
@MickL 

It uses the same components from this repo, I copied them to my new project. 

I discussed this solution with @Flowko. It's super hard to maintain and support Vue. That's why we now have a Nuxt only solution. 

If anyone has the knowledge and motivation to develop & maintain a Vue solution, that would be amazing. I will remove my module once we have that.

**Comment by MickL** [comment]
@Mokkapps Personally I use vue-email in my backend only (no Nuxt involved, only Vue) so I am the perfect example of people not benefitting from whatever you implement into nuxt-email-renderer. Not sure if it is a good solution to have two separate libraries from now on.

**Comment by Mokkapps** [comment]
@MickL Yes, I definitely agree that it would be ideal if we would have only one library that is maintained and can be used for Vue & Nuxt.

But as we don't have anyone that can maintain and further develop this library, I'm happy that I have a Nuxt solution that I can work with and adjust for my needs. 

I'm using this new module for my products (Weekly Vue News & Nuxt Starter Kit), so I'm highly interested in keeping my module up to date and maintained.
